handbook/openid-connect/openid-connect.md

```diff
@@ -316,3 +316,17 @@ The following sources can be used to troubleshoot a failed connection.
     SystemLink uses log rotation therefore the latest logs may be in one of the numbered `error.log` files.
 
 **Returned Claims:** See [**Viewing Claims Returned by a Provider**](#viewing-claims-returned-by-a-provider).
+
+### Cache Entry Size
+
+The OpenID Connect module stores information in a shared memory cache. If a cache entry is too large, users will see an "Internal Server Error" when attempting to log in, and the NI Web Server logs will contain the following entries
```

Comment:
Can we add a mention that you can get into this state if you are returning a large number of claims?

Do we know of any other potential ways we could get into this state?

Reply:
Or claims with large values